diff --git a/pyproject.toml b/pyproject.toml
index c065ac7..9e88fb3 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -79,7 +79,8 @@ dependencies = [
     "packaging==26.2",
     "pexpect==4.9.0",
     "platformdirs==4.9.6",
-    "protobuf==6.33.6",
+    # Needed until https://github.com/conda-forge/protobuf-feedstock/issues/274  is fixed
+    "protobuf",
     "ptyprocess==0.7.0",
     "pyasn1==0.6.3",
     "pyasn1-modules==0.4.2",
@@ -100,7 +101,8 @@ dependencies = [
     "requests==2.33.1",
     "rich==15.0.0",
     "rpds-py==0.30.0",
-    "secretstorage==3.5.0 ; sys_platform == 'linux'",
+    # Needed until https://github.com/conda-forge/secretstorage-feedstock/pull/27 is fixed
+    "secretstorage",
     "sentry-sdk==2.63.0",
     "six==1.17.0",
     "smmap==5.0.3",
